crates/flipperzero/src/furi_alloc.rs

```diff
@@ -0,0 +1,57 @@
+use core::alloc::{GlobalAlloc, Layout};
+use core::ffi::c_void;
+
+use flipperzero_sys::c_string;
+use flipperzero_sys::furi;
+
+extern "C" {
+    fn aligned_free(p: *mut c_void);
+    fn aligned_malloc(size: usize, align: usize) -> *mut c_void;
+    fn memmgr_get_total_heap() -> usize;
+    fn memmgr_get_free_heap() -> usize;
+    fn memmgr_get_minimum_free_heap() -> usize;
+}
```

Comment:
I have some [bindings for `memmgr` functions on the `0.2-dev` branch](https://github.com/dcoles/flipperzero-rs/blob/0.2-dev/crates/sys/src/furi/memmgr.rs), but it's fine use these ones until it's merged.